mediator/med_phases_restart_mod.F90

```diff
@@ -52,7 +52,8 @@ subroutine med_phases_restart_write(gcomp, rc)
     type(ESMF_TimeInterval)    :: timediff       ! Used to calculate curr_time
     type(ESMF_Alarm)           :: alarm
     type(ESMF_Calendar)        :: calendar
-    character(len=64)          :: currtimestr, nexttimestr
+    character(len=64)          :: currtimestr
```

Comment:
Please don't hard code string lengths - use one of the already defined constants.

Reply:
Done

Reply:
Done.